### PR TITLE
added alt atributes to all non decorative images

### PR DESCRIPTION
**Pull request by Granoffel**

### CODE REVIEW

File: app/src/components/search/AreaSelect.vue [hunk 1]
```diff
@@ -11,12 +11,12 @@
     @change="onInputChange"
   >
     <template v-slot:item="{ item }">
-      <img class="areaImage" :src="item.img" />
+      <img class="areaImage" alt="areaImage" :src="item.img" />
```
Comment (theErikss0n) [inline app/src/components/search/AreaSelect.vue:14]: "areaImage" is not descriptive. We should set the alt-tag to something like "Deutsche Flagge" and "Erdkugel" depending on the selectecd item.

File: app/src/components/search/AreaSelect.vue [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
       <v-spacer />
       <span>{{ item.text }}</span>
     </template>
     <template v-slot:selection="{ item }">
-      <img class="areaImageSelected" :src="item.img" />
+      <img class="areaImageSelected" alt="areaImageSelected" :src="item.img" />
```
Comment (theErikss0n) [inline app/src/components/search/AreaSelect.vue:19]: Like above, we should find a better fitting alt-tag.

**Review by theErikss0n — CHANGES_REQUESTED** [review]
The carousel alt-tags look good!

We should also add alt-tags for our image cards, as they are also non-decorative.

**Review by theErikss0n — CHANGES_REQUESTED** [review]
Area select tags are good now!

We still need alt-tags for the image cards.

**Comment by Granoffel** [comment]
The image cards a decorative. Image is set as background. Crawler recognise these as decorative.
Cards will be redesigned in #480

**Review by theErikss0n — APPROVED** [review]

**Pull request merged**